File: Common/Entity/src/main/java/it/ey/dto/UtenteRuoloPaDTO.java
package it.ey.dto;

import lombok.*;

import java.time.LocalDate;
import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class UtenteRuoloPaDTO {

    private String id;

    private String codiceFiscale;
    private String nome;
    private String cognome;
    private String email;
    private String numeroTelefono;
    private LocalDate dataNascita;
    private String luogoNascita;
    private List<RuoloUtenteDTO> ruoli;
    private List<UtentePaDTO> codicePA;
    private List<UtenteRuoliPaSezioneDTO> sezioni;
}

File: dfp-piao-fe/src/app/shared/models/classes/utente-ruolo-pa-dto.ts
import { UtenteRuoliPaSezioneDTO } from './utente-ruoli-pa-sezione-dto';
import { UtenteRuoloDTO } from './utente-ruolo-dto';
import { UtentePaDTO } from './utente-pa-dto';

export class UtenteRuoloPaDTO {
  id?: string;
  codiceFiscale?: string;
  nome?: string;
  cognome?: string;
  email?: string;
  numeroTelefono?: string;
  dataNascita?: string;
  luogoNascita?: string;
  ruoli?: UtenteRuoloDTO[];
  codicePA?: UtentePaDTO[];
  sezioni?: UtenteRuoliPaSezioneDTO[];
}
